pkg/apis/proxy/proxy.go

```diff
@@ -177,22 +178,23 @@ func handleProxyCommon(p *SprayProxy, c *gin.Context) {
 			continue
 		}
 		newRequest.Header = copy.Request.Header
-		// currently not distinguishing between requests we send and requests that return without error
-		metrics.IncForwardedCount(backendURL.Host)
 
 		// for response time, we are making it "simpler" and including everything in the client.Do call
 		start := time.Now()
 		resp, err := client.Do(newRequest)
 		responseTime := time.Now().Sub(start)
-		metrics.AddForwardedResponseTime(responseTime.Seconds())
 		// standartize on what ginzap logs
 		zapBackendFields = append(zapBackendFields, zap.Duration("latency", responseTime))
 		if err != nil {
+			fwdErr = "http client error"
```

Comment:
minimally append the HTTP response code to this string

Ideally there would be some relevant portion of the error message that would be both informative but also short and consistent, but I suspect that will not be the case.  In any event, if need be we can iterate on this over time, but most likely the response code is sufficient for now.

Reply:
There is no HTTP response code in this case as the HTTP server is not reached. I also looked into the http.Client.Do() code and there are a bunch of different errors and I did not want such variety of labels, so went for the hardcoded one.

Reply:
OK, then let's have a "problem" label that either is set to
- "none", meaning we got a 200, 201, whatever valid HTTP RC we expect
- "unexpected-rc-<any http rc outside of the valid HTTP rc's>
- "http-client-error" for the http server is not even reached case 

IIRC having spaces in the prometheus label values is not recommended

Reply:
OK. I came with the "none", "http-error" (rc >=400) and "non-http-error".